test/lib/tasks/institutions_export_task_test.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+module Otis
+  class InstitutionsExportTaskTest < ActiveSupport::TestCase
```

Comment:
Probably would be worth checking the content of the files (at least one line of it) -- do we have fixture data in the institutions table?